server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestClearVotingTombstonesAction.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.rest.action.admin.cluster;
+
+import org.elasticsearch.action.admin.cluster.configuration.ClearVotingTombstonesAction;
+import org.elasticsearch.action.admin.cluster.configuration.ClearVotingTombstonesRequest;
+import org.elasticsearch.client.node.NodeClient;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.rest.BaseRestHandler;
+import org.elasticsearch.rest.RestController;
+import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.rest.action.RestToXContentListener;
+
+import java.io.IOException;
+
+public class RestClearVotingTombstonesAction extends BaseRestHandler {
+
+    public RestClearVotingTombstonesAction(Settings settings, RestController controller) {
+        super(settings);
+        controller.registerHandler(RestRequest.Method.DELETE, "/_cluster/withdrawn_votes", this);
+    }
+
+    @Override
+    public String getName() {
+        return "clear_voting_tombstones_action";
+    }
+
+    @Override
+    protected RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client) throws IOException {
+        ClearVotingTombstonesRequest req = new ClearVotingTombstonesRequest();
+        req.setWaitForRemoval(false);
```

Comment:
Apologies, I meant this should be a parameter `?waitForRemoval=false` - both options are useful in different circumstances. The default of `true` is the usual case, but in this particular test we should set it to `false` because the node is still present.

Now that I've written that, I think it'd be good to test both cases:

1. create a 3-node cluster, add a tombstone, then clear them (`?waitForRemoval=false`) <- today's test
2. create a 3-node cluster, add a tombstone, shut the corresponding node down, then clear the tombstones (`?waitForRemoval=true`)

Reply:
Looking at other APIs, it should be `wait_for_removal` not `waitForRemoval`.